config.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  enable-dynamic-allocation:
+    type: boolean
+    description: |
+      Enable dynamic allocation of pods for autoscaling Spark jobs.
+      Is equivalent to the following Spark configuration:
```

Comment:
```suggestion
      Is equivalent to the following Spark configurations:
```

Reply:
Done in e92513ff865dd8f82bceb8128851982dce8eaf2c, thanks